sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_joint_pos.py

```diff
@@ -33,11 +33,12 @@
 # It is recommended to run this script in simulation first.
 
 import rospy
+import sys
 from sr_robot_commander.sr_arm_commander import SrArmCommander
 from sr_robot_commander.sr_hand_commander import SrHandCommander
 from sr_robot_commander.sr_robot_commander import SrRobotCommander
 
-rospy.init_node("right_hand_arm_joint_pos", anonymous=True)
+rospy.init_node("right_hand_arm_joint_pos", anonymous=True, disable_signals=True)
```

Comment:
why are we disabling signals here?

Reply:
oh yea sorry was trying something. removing now